content/en/apps/guides/updates/preparing-for-4.0.md

```diff
@@ -0,0 +1,162 @@
+
+We'll [use `jq` to filter](https://stedolan.github.io/jq/) out the unrelated metrics.  Be sure to replace `CHT-URL-HERE` with your production CHT URL:
+
+```shell
+curl -s https://CHT-URL-HERE/api/v2/monitoring\?connected_user_interval\=30 | \
+  jq '. | {connected_users}'
+```
+
+This call will show the JSON with active users for the last 30 days:
+
+```json
+{
+  "connected_users": {
+    "count": 80
+  }
+}
+```
+
+### Users in need up upgrade
+
+In the above table we can see there's 4 users in the most recent month that are on version `v0.11.0-webview`. Let's find their username so we can follow up with them directly.  This query is hard coded for the current year (`2022`), the current month (`3`) and the version `v0.11.0-webview`. Be sure to update the query to fit your needs according to which month, year and version you need to search for:
+
+```sql
+select 
+	distinct doc#>>'{metadata,user}' as user, 
+	doc#>>'{device,deviceInfo,app,version}' as cht_android_version
+from 
+	couchdb_users_meta 
+where 
+	doc#>>'{type}' = 'telemetry'  
+	and doc#>>'{device,deviceInfo,app,version}' is not null 
+	and doc#>>'{metadata,year}' = to_char(current_date, 'yyyy')
+	and doc#>>'{metadata,month}' = to_char(current_date, 'FMMM')
+	and doc#>>'{device,deviceInfo,app,version}' = 'v0.11.0-webview'
+```
+
+In this case only one user is found:
+
+```
+user|cht_android_version|
+----+-------------------+
+adj |v0.11.0-webview    |
+```
+
+A twist on this query is to remove the version filter. Do not remove the year and month filter as you will get duplicate values for users who have upgraded over time:
+
+```sql
+select 
+	distinct doc#>>'{metadata,user}' as user, 
+	doc#>>'{device,deviceInfo,app,version}' as cht_android_version
+from 
+	couchdb_users_meta 
+where 
+	doc#>>'{type}' = 'telemetry'  
+	and doc#>>'{device,deviceInfo,app,version}' is not null 
+	and doc#>>'{metadata,year}' = to_char(current_date, 'yyyy')
+	and doc#>>'{metadata,month}' = to_char(current_date, 'FMMM')
+```
+
+This will give a list of every user and the version they're running as of the current month and year:
+
+```
+user            |cht_android_version|
+----------------+-------------------+
+moh-west-chw-1  |v1.0.1-4           |
+moh-west-chw-12 |v1.0.1-4           |
+moh-west-chw-14 |v1.0.1-4           |
+moh-west-chw-3  |v1.0.1-4           |
+moh-west-chw-3  |v1.0.1-5           |
+moh-west-chw-4  |v1.0.1-5           |
+moh-west-chw-5  |v1.0.1-4           |
+moh-west-chw-6  |v1.0.1-5           |
+moh-west-chw-7  |v1.0.1-5           |
+moh-west-chw-8  |v1.0.1-4           |
+adj             |v0.11.0-webview    |
+```
+
+## CHT Conf
+
+[CHT Conf](https://github.com/medic/cht-conf/) has been upgrade to be aware of forms written to work in CHT 3.x that may not work in CHT 4.x.  By upgrading, you will be sure to catch any forms in need up fixing when pushing to dev instances as outlined below.
```

Comment:
```suggestion
[CHT Conf](https://github.com/medic/cht-conf/) has been upgraded to be aware of forms written to work in CHT 3.x that may not work in CHT 4.x.  Upgrading, can help you identify forms in need of fixing when pushing to dev instances as outlined below.
```

I think we should soften this a bit since cht-conf will definitely not be able to catch _all_ the potential form issues...